packages/manager/src/features/Support/SupportTicketDetail/SupportTicketDetail.tsx

```diff
@@ -79,11 +82,7 @@ export const SupportTicketDetail = () => {
           crumbOverrides: [
             {
               linkTo: {
-                pathname: `/support/tickets`,
-                // If we're viewing a `Closed` ticket, the Breadcrumb link should take us to `Closed` tickets.
-                search: `type=${
-                  ticket.status === 'closed' ? 'closed' : 'open'
-                }`,
+                pathname: `/support/tickets/${ticket.status}`,
```

Comment:
I don't think we want to try to route to `/support/tickets/new`?

```suggestion
                pathname: `/support/tickets/${ticket.status === 'new' ? 'open' : ticket.status}`,
```

https://github.com/user-attachments/assets/2b960c45-5a36-4b9b-a230-875bb1544002

Reply:
nice catch! correct, although `/open` would not open the dialog (which i assume would be the intended behavior of `/new`). To that effect I am adding a /new route which redirects to `/support/tickets?dialogOpen=true` - this will preserve the behavior shall anyone still link to `/new`